tests/no_inst_fleet_unit_tests.py

```diff
@@ -73,26 +75,9 @@ def get_cursor(file_name):
             "name": "source"
         },
             {
-            "config": {"Sink": {"in_commods": {"val": "spentfuel"},
+            "config": {"Sink": {"in_commods": {"val": "fuel"},
                                 "max_inv_size": "1e6"}},
             "name": "sink"
-        },
-            {
-            "config": {
-                "Reactor": {
-                    "assem_size": "1000",
-                    "cycle_time": "1",
-                    "fuel_incommods": {"val": "fuel"},
-                    "fuel_inrecipes": {"val": "fresh_uox"},
-                    "fuel_outcommods": {"val": "spentfuel"},
-                    "fuel_outrecipes": {"val": "spent_uox"},
-                    "n_assem_batch": "1",
-                    "n_assem_core": "3",
-                    "power_cap": "1000",
-                    "refuel_time": "1",
-                }
-            },
-            "name": "reactor"
```

Comment:
You're removing the reactor from this file?

Reply:
It is not necessary for these tests as they only test fuel demand and supply, but I will add it back into the file for future tests